ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/ProfileChecksPage.tsx

```diff
@@ -86,13 +95,19 @@ function ProfileChecksPage() {
             <PageTitle title="Compliance coverage - Profile checks" />
             <CoveragesPageHeader />
             <PageSection>
+                {!disclaimerAccepted && (
```

Comment:
At least in this PatternFly context, what do you think if rendering component takes responsibility for flex layout instead of `className` prop within element?
`<Flex direction={{ default: 'column' }} spaceItems={{ default: 'spaceItemsMd' }}>`

Reply:
That's an option I hadn't considered. I didn't like writing out the className prop...

Reply:
Updated and I like it, but I also realized the structure is going to have to change anyway when the benchmark tabs are introduced